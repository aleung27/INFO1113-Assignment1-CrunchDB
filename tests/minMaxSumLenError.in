SET a 4 5
MIN A
MAX A
SUM A
LEN A
POP a
POP a
MIN a
MAX a
SUM a
LEN a
BYE
