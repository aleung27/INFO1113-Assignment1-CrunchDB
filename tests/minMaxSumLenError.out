> ok

> no such key

> no such key

> no such key

> no such key

> 4

> 5

> nil

> nil

> nil

> 0

> bye
